### ISSUE TITLE
apm install gets 0.2.1 instead of 1.0.0

### ISSUE THREAD

**Issue by chriskrycho** [Alchiadus/base16-syntax#7]
It's entirely possible this is on my end, but nothing I do changes the fact that when I run `apm install base16-syntax` it installs 0.2.1 instead of 1.0.0 *unless* I run it as `apm install base16-syntax@1.0.0`. I'm running Atom 0.177.0.

**Comment by Alchiadus** [comment]
Thank you for reporting this.

On my main machine I did not get the update to 1.0.0 and I thought this was due to the fact I symlink the dev version to Atom's Dev directory and that it somehow prevented me from getting the update in Atom's non-dev version (unlikely, this workflow has always worked fine before). I tested installing the package on a fresh install of Atom on another machine [Windows 8, Atom 0.175.0] and it correctly installed 1.0.0, so I thought it was on my end, somehow. Together with your report, I no longer believe this is the case. I'll look into what might cause the issue.
